common/src/main/resources/assets/wynntils/lang/en_us.json

```
@@ -1175,6 +1175,10 @@
   "feature.wynntils.perCharacterGuildContribution.hideContributionMessage.description": "Hides the contribution message sent after automatically applying your contribution amount.",
   "feature.wynntils.perCharacterGuildContribution.hideContributionMessage.name": "Hide Auto Contribution Message",
   "feature.wynntils.perCharacterGuildContribution.name": "Per Character Guild Contribution",
+  "feature.wynntils.personalStorageUtilities.LockedQuickJumpColor.description": "What color should the QuickJump button be when not unlocked/visited?",
+  "feature.wynntils.personalStorageUtilities.LockedQuickJumpColor.name": "Locked QuickJump Color",
+  "feature.wynntils.personalStorageUtilities.SelectedQuickJumpColor.description": "What color should the QuickJump button be when selected?",
+  "feature.wynntils.personalStorageUtilities.SelectedQuickJumpColor.name": "Selected QuickJump Color",
```

```suggestion
  "feature.wynntils.personalStorageUtilities.LockedQuickJumpColor.description": "What color should the Quick Jump button's text be when not unlocked/visited?",
  "feature.wynntils.personalStorageUtilities.LockedQuickJumpColor.name": "Locked Quick Jump Text Color",
  "feature.wynntils.personalStorageUtilities.SelectedQuickJumpColor.description": "What color should the Quick Jump button's text be when selected?",
  "feature.wynntils.personalStorageUtilities.SelectedQuickJumpColor.name": "Selected Quick Jump Text Color",
```

And remember to change the config name here when applying the change from my previous comment